### ISSUE TITLE
apollo:codegen fails on macOS & Windows

### ISSUE THREAD

**Issue by kremalicious** [oceanprotocol/market#359]
Thanks to #358, we know that our `npm run apollo:codegen` fails on macOS & Windows:

Windows:
<img width="904" alt="Screen Shot 2021-02-01 at 12 25 59" src="https://user-images.githubusercontent.com/90316/106452789-a61a3f80-6488-11eb-9b1e-bc8b32bc6272.png">
https://github.com/oceanprotocol/market/runs/1806089346?check_suite_focus=true#step:8:1

macOS:
<img width="955" alt="Screen Shot 2021-02-01 at 12 26 48" src="https://user-images.githubusercontent.com/90316/106452874-c34f0e00-6488-11eb-8d1e-088d5fe73ed7.png">
https://github.com/oceanprotocol/market/runs/1806089315?check_suite_focus=true#step:8:2

On Ubuntu all works, this is how a success looks like:
<img width="932" alt="Screen Shot 2021-02-01 at 12 29 13" src="https://user-images.githubusercontent.com/90316/106453152-28a2ff00-6489-11eb-8f2d-d5d399b47c39.png">

**Comment by kremalicious** [comment]
This `sed` part seems the problem in both cases:

```bash
find ./src/@types/apollo -type f -exec sed -i -r 's/_([a-z])/\\U\\1/gi' {} \\;
```

Think Windows in general doesn't like too many quotes in commands which could lead to incorrect syntax error. Error on macOS seems this: https://stackoverflow.com/questions/28072190/sed-error-1-not-defined-in-the-re-on-macosx-10-9-5

@mihaisc what is the command supposed to do? Maybe we can tweak or rewrite

**Comment by mihaisc** [comment]
It fixes the format of the props, it converts to camel case. By default it creates classes/files/props like Pool_poolShares

**Comment by mihaisc** [comment]
Maybe try something like this https://github.com/75lb/renamer/wiki/examples#regular-expression-examples? Seems to work on all os

**Comment by kremalicious** [comment]
root of problem seems to be the difference between GNU sed and POSIX sed, where the latter does not support Extended Regular Expressions (ERE) we use with the `-r` flag https://unix.stackexchange.com/a/499307/125382

**Comment by mihaisc** [comment]
But what do you do on windows?

**Comment by kremalicious** [comment]
well, I tried. But then I saw we only run into that `PascalCase` issue on one import. So seems better way to remove that renaming altogether in https://github.com/oceanprotocol/market/pull/358/commits/5a7887ae3ef893c99efaf03300436b72d70e0808 and wait for resolve of https://github.com/apollographql/apollo-tooling/issues/1414

**Comment by mihaisc** [comment]
When i first added it, this issue was on almost all the interfaces and files had an underscore, but if you tested and it happens in one place lets remove it

**Comment by kremalicious** [comment]
think I got it now, working on ubuntu & macOS. For Windows, next issue was `mkdir` which exists on Windows but it needs different slashes for file path than on UNIX systems. Solved by using a cross-platform package for that.

But for whatever reason in Node.js v14 Windows can't pick up the generated typings, no problem at all on Node.js v15  https://github.com/oceanprotocol/market/runs/1806974740?check_suite_focus=true
